### PR TITLE
Analysis.

### PR DESCRIPTION
**Pull request by vulasau**

### CODE REVIEW

File: Unite.Web/Client/src/analysis/store/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    analyses: new Map(),
+    token: null,
+  };
+
+  getters = { ...getters };
+
+  actions = { ...actions };
+}
+
+/*
```
Comment (DGitRich) [inline Unite.Web/Client/src/analysis/store/index.js:17]: Can this commented block be deleted?

File: Unite.Web/Client/src/cohorts/CohortsPage.vue [hunk 2]
```diff
@@ -31,11 +31,15 @@
 
   <!-- Page content -->
   <div class="col q-gutter-y-sm">
-    <div class="row">
+    <!-- <div class="row">
```
Comment (DGitRich) [inline Unite.Web/Client/src/cohorts/CohortsPage.vue:34]: Can this be deleted?

File: Unite.Web/Client/src/cohorts/CohortsPage.vue [hunk 3]
```diff
@@ -107,23 +113,29 @@ export default {
...
+      return this.domains?.flatMap(domain => domain.cohorts.filter(cohort => cohort.selected));
     }
   },
 
   mounted() {
     this.domains = this.getDomainOptions(this.availableDomains);
     this.domain = this.domains.find(domain => domain.name == this.$route.params.domain) || this.domains[0] || null;
-    this.cohort = this.domain?.cohorts[0];
+    this.cohort = this.domain?.cohorts[0] || null;
+    // this.loadDomainData(this.domain);
```
Comment (DGitRich) [inline Unite.Web/Client/src/cohorts/CohortsPage.vue:127]: Commented code

**Review by DGitRich — APPROVED** [review]

File: Unite.Web/Client/src/cohorts/CohortsPage.vue (hunk 2 above)
Comment (vulasau) [reply Unite.Web/Client/src/cohorts/CohortsPage.vue:34]: We need to leave it, as I don't know yet, how breadcrumbs should be displayed.

**Pull request merged**